packages/wdio-protocols/protocols/saucelabs.json

```diff
@@ -199,5 +209,25 @@
         "description": "Returns true if any metrics have a regression."
       }
     }
+  },
+  "/session/:sessionId/sauce/ondemand/performance/scroll": {
+    "POST": {
+      "command": "jankinessCheck",
```

Comment:
shouldn't command name be something like getScrollPerformance ?

Reply:
We are not sure with the naming yet, jankiness can also be captured without scrolling. This command just does scrolling as one method. Overall the metrics that are return give information about how smooth the app behaved meaning how janky it was.